ethportal-api/src/types/execution/header.rs

```diff
@@ -195,6 +197,73 @@ impl PartialEq for Header {
     }
 }
 
+impl From<Header> for RpcHeader {
```

Comment:
Could you add a docstring comment here explaining why we need this `RpcHeader` type.